.tx/config

```diff
@@ -9,4 +9,5 @@ trans.fr = locale/fra/LC_MESSAGES/django.po
 trans.es = locale/es/LC_MESSAGES/django.po
 trans.hi = locale/hin/LC_MESSAGES/django.po
 trans.pt = locale/por/LC_MESSAGES/django.po
+trans.sw = locale/sw/LC_MESSAGES/django.po
```

Comment:
This isn't strictly necessary, as this is the default directory transifex searches for anyways, but I like being explicit about it (I actually just did the [same change](https://github.com/dimagi/commcare-hq/commit/a9261f731d4a36f83d9ca49dba9fc9e63e5ebbaf)).

The language also needs to be added through the transifex UI, but someone did that a while ago.  The only other thing that's a bit tricky is that if you edit the `.po` files locally, you also need to update the timestamp in the metadata at the top and push that to transifex or else transifex will ignore the edits, and they'll get overwritten next time someone pulls translations.  I pulled in the old translations which had been manually added from an old commit [here](https://github.com/dimagi/commcare-hq/commit/2820c5d98a165718bb1a8eb347eca9324257edde).

This PR tags the strings for translation again, so next time someone runs `makemessages`, it should take the old translations (currently stored but not used) and apply them to these strings.